src/ImageQuilting.jl

```diff
@@ -21,12 +21,15 @@ using RecipesBase
 using SparseArrays: spzeros
 using Statistics: mean, std
 using Random
+using CUDA
+using ComputationalResources
```

Comment:
Please move these two lines to before RecipesBase, and listing CUDA after the ComputationalResources